<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadingLibrary</class>
 <widget class="QDialog" name="LoadingLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loading Database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/database.png</normaloff>
    <normalon>icons/database.png</normalon>icons/database.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QProgressBar" name="library_loading_progress_bar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="library_loading_task_label">
     <property name="text">
      <string>currentlyPerforming</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
